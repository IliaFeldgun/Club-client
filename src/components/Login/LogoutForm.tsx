import React from 'react'
import ClubSession from '../../utils/ClubSession'

export default class LogoutForm extends React.PureComponent {
    handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
        ClubSession.killSession()
    }
    render() {
        const buttonClass = "form-button"
        return (
            <button className={buttonClass} onClick={this.handleClick}>
            <span>
                Logout
            </span>
        </button>
        )
    }
}