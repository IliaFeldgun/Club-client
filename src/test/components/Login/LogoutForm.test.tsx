import React from 'react'
import { mount, render, shallow } from 'enzyme'
import LogoutForm from '../../../components/Login/LogoutForm'

describe('Testing LogoutForm', () => {
    test('component renders', () => {
        const component = shallow(<LogoutForm  />)
        expect(component).toMatchSnapshot()
    })
    
})