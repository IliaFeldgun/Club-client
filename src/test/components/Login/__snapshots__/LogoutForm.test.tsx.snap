// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing LogoutForm component renders 1`] = `
<button
  className="formButton-0-2-1"
  onClick={[Function]}
>
  <span>
    Logout
  </span>
</button>
`;
